mso/resource_mso_site.go

```diff
@@ -79,6 +80,7 @@ func resourceMSOSite() *schema.Resource {
 			"urls": &schema.Schema{
 				Type:     schema.TypeList,
 				Optional: true,
+				Computed: true,
```

Comment:
Would it be a good idea to change the "location" attribute to Computed as well?

Reply:
changed.